spec/controllers/api/v1/providers_controller_spec.rb

```diff
@@ -20,5 +20,16 @@
 
       get :index
     end
+
+    it 'renders a 400 when the changed_since param is not valid' do
+      allow(controller).to receive(:authenticate)
+
+      get :index, params: { changed_since: '2019' }
```

Comment:
[This post](https://www.myintervals.com/blog/2009/05/20/iso-8601-date-validation-that-doesnt-suck/) reckons that `2019` is a valid ISO8601 date...

Reply:
Perhaps rather than tying ourselves up in knots, we should specify a more strict interface...

Reply:
It is a valid ISO8601 date, it is not a valid ISO8601 UTC datetime timestamp. This stuff is very pedantic. I think supplying a good example is enough to disambiguate.

Reply:
Fair enough...